_static/custom.css

```diff
@@ -5,17 +5,17 @@
  * Custom stylesheet
  *
  */
-@import url('https://fonts.googleapis.com/css?family=Roboto:100,300,300i,400,500,700,900');
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,300,300i,400,500,700,900');  /* Ignore CSSLintBear */
```

Comment:
these additions are because [this repo's .coafile](https://github.com/coala/documentation/blob/master/.coafile) doesnt have csslint enabled.

And this PR would allow that to be added to .coafile.